Let SymlinkProgram run depmod and let the recipe backup original modules
--- kvm-68/kernel/Makefile	2008-05-02 13:20:13.000000000 +0200
+++ kvm-68.new/kernel/Makefile	2008-05-12 19:03:21.000000000 +0200
@@ -76,11 +76,6 @@
 install:
 	mkdir -p $(DESTDIR)/$(INSTALLDIR)
 	cp *.ko $(DESTDIR)/$(INSTALLDIR)
-	for i in $(ORIGMODDIR)/drivers/kvm/*.ko \
-		 $(ORIGMODDIR)/arch/$(ARCH_DIR)/kvm/*.ko; do \
-		if [ -f "$$i" ]; then mv "$$i" "$$i.orig"; fi; \
-	done
-	/sbin/depmod -a
 
 tmpspec = .tmp.kvm-kmod.spec
 
